### PR TITLE
odoo chart

### PR DESCRIPTION
**Pull request by sameersbn**

### CODE REVIEW

**Comment by viglesiasce** [comment]
@k8s-bot ok to test

**Comment by viglesiasce** [comment]
@k8s-bot ok to test

File: stable/odoo/requirements.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+dependencies:
+- name: postgresql
+  version: 0.2.1
```
Comment (viglesiasce) [inline stable/odoo/requirements.yaml]: Change this to point to 0.3.0 which has the fix for imagePullPolicy on the init container.

**Review by viglesiasce — CHANGES_REQUESTED** [review]

**Comment by sameersbn** [comment]
Completed the suggested changes, however I would like this chart to be merged after #318

**Pull request merged**